packages/techdocs-common/src/helpers.ts

```diff
@@ -107,7 +107,7 @@ export const getGitRepositoryTempFolder = async (
     // fs.realpathSync fixes a problem with macOS returning a path that is a symlink
     fs.realpathSync(os.tmpdir()),
     'backstage-repo',
-    parsedGitLocation.source,
+    parsedGitLocation.resource,
```

Comment:
Works 👌

Reply:
Oh gosh this mistake has happened many times. Very confusing choice of field name on their part